lib/p2p/NodeList.ts

```diff
@@ -96,6 +100,14 @@ class NodeList {
         await node.save();
         return true;
       }
+
+      if (node.lastAddress && addressUtils.areEqual(address, node.lastAddress)) {
```

Comment:
adding a small comment here would be nice

Reply:
sure